### PR TITLE
Add gretchenfrage as a maintainer

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

**Review by gretchenfrage — APPROVED** [review]
Hooray!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
-* @djc @Ralith
+* @djc @Ralith @gretchenfrage
 /quinn-udp @djc @Ralith @mxinden
```
Comment (Ralith) [inline .github/CODEOWNERS:2]: Do we need to modify line 2 as well, or is it redundant as-is?

**Review by Ralith — APPROVED** [review]
Happy to have you!

File: .github/CODEOWNERS (hunk 1 above)
Comment (gretchenfrage) [reply .github/CODEOWNERS:2]: I think it might be redundant as-is?

Comment (gretchenfrage) [reply .github/CODEOWNERS:2]: GitHub's official example seems to indicate it's redundant as-is. https://github.com/github/docs/blob/main/content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md#example-of-a-codeowners-file

Comment (Ralith) [reply .github/CODEOWNERS:2]: Cool, we can fix that in a followup then.

Comment (Ralith) [reply .github/CODEOWNERS:2]: Ah, then it's not redundant, and we should probably add @gretchenfrage there too?

Comment (djc) [reply .github/CODEOWNERS:2]: Yup!

**Pull request merged**